plugins/main/public/components/agents/fim/inventory/fileDetail.tsx

```diff
@@ -411,61 +322,6 @@ export class FileDetails extends Component {
     );
   }
 
-  renderFileDetailsPermissions(value) {
-    if (
-      ((this.props.agent || {}).os || {}).platform === 'windows' &&
-      value &&
-      value !== '-'
-    ) {
```

Comment:
**question**: Where is the condition moved? I do not see it.

Reply:
It's on fim-data. It's only use on 

```
{
    field: 'perm',
    columns: null,
    details: {
      name: 'Permissions',
      icon: 'lock',
      link: false,
      transformValue: value => renderFileDetailsPermissions(value),
    },
    suggestion: null,
  },
```